src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,49 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int totalSupply = 0;
+        int totalBuy = 0;
+
+        try (BufferedReader br = new BufferedReader(new FileReader(fromFileName))) {
+            String line;
+            while ((line = br.readLine()) != null) {
+                String[] parts = line.split(",");
+                if (parts.length != 2) {
+                    continue;
+                }
+                String operationType = parts[0].trim();
+                int amount;
+                try {
+                    amount = Integer.parseInt(parts[1].trim());
+                } catch (NumberFormatException e) {
+                    continue;
+                }
```

Comment:
Catching `NumberFormatException` and continuing without logging or handling the error might lead to silent failures. Consider logging the error or handling it in a way that informs the user of the malformed data.